core/src/main/java/io/syndesis/core/Tokens.java

```diff
@@ -56,6 +68,65 @@ public static boolean isTokenExpired(String token) {
         }
     }
 
+    private static String getIssuer(String token) {
+        TokenVerifier verifier = TokenVerifier.create(token);
+        try {
+            return verifier.getToken().getIssuer();
+        } catch (VerificationException e) {
+            throw SyndesisServerException.launderThrowable(e);
+        }
+    }
+
+    public static String getProviderToken(String providerId) {
+        String keycloakTokenAsString = getAuthenticationToken();
+
+        String issuer = getIssuer(keycloakTokenAsString);
```

Comment:
Question: The issuer of the keycloak token is always the same as the API Url to the keycloak server or is this just a convention ? And if it is a convention, one that can be taken for granted or something we need to document (e.g. as comment in the keycloak configmap) ?